.github/workflows/tests.yml

```diff
@@ -37,15 +37,18 @@ jobs:
             - 'Cargo.toml'
             - 'Cargo.lock'
             - '.rustfmt.toml'
+            - '.github/workflows/tests.yml'
```

Comment:
At what point are we reinventing `make` here? (Pedantry: make wouldn't help us here because there's no "I have run tests" file whose timestamp can be used to decide what needs to be run.)

Reply:
🤷 Idk, this is always going to be a bit manual though, isn't it? (I hate CI.)